examples/apps/attributable-map/src/modelContainerRuntimeFactoryWithAttribution.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IContainer,
+	IContainerContext,
+	IRuntime,
+	IRuntimeFactory,
+} from "@fluidframework/container-definitions";
+import { IContainerRuntimeOptions, ContainerRuntime } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { NamedFluidDataStoreRegistryEntries } from "@fluidframework/runtime-definitions";
+import {
+	mixinAttributor,
+	createRuntimeAttributor,
+	IProvideRuntimeAttributor,
+} from "@fluid-experimental/attributor";
+import { FluidObject } from "@fluidframework/core-interfaces";
+import { makeModelRequestHandler } from "@fluid-example/example-utils";
+
+const containerRuntimeWithAttribution = mixinAttributor(ContainerRuntime);
+
+/**
+ * ModelContainerRuntimeFactoryWithAttribution is an abstract class that gives a basic structure for container runtime initialization with attributor enabled.
+ * It also requires a createModel method to returns the expected model type.
+ */
+export abstract class ModelContainerRuntimeFactoryWithAttribution<ModelType>
```

Comment:
This doesn't seem appropriate - it's not realistic to require customers to use a different specialized runtime factory for each feature they want to enable.  What would a normal customer do?